CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # CHANGELOG
 
 ## Next Release
+- Bump threeten-extra to v1.7.2
```

Comment:
Looks like we should put this under the `Internal` below, right?

Also, I'm guessing that the rest of the changes have been released already, but the threeten-extra update is yet to be released?

Reply:
yes, rest of the changes have been released already.